pkg/services/sqlstore/alert.go

```diff
@@ -295,6 +296,34 @@ func PauseAlert(cmd *m.PauseAlertCommand) error {
 	})
 }
 
+func PauseDashboardAlerts(cmd *m.PauseDashboardAlertsCommand) error {
+	return inTransaction(func(sess *DBSession) error {
+		if cmd.DashboardId == 0 {
+			return fmt.Errorf("command contains no dashboard id")
+		}
+
+		var buffer bytes.Buffer
+		params := make([]interface{}, 0)
```

Comment:
Avoid using make for creating empty slices. Instead, create an empty slice

```go
var params []interface{}
``` 

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices